src/user/workflow_monitor_debug.rst

```diff
@@ -0,0 +1,250 @@
+the workflow. There are different ways how such a list of tasks could be
+retrieved. One of the ways is to drill down using the link provided in
+the WorkFlow progress view described earlier. Another way is to use the
+workflow name, e.g.::
+
+    https://usdf-panda-bigmon.slac.stanford.edu:8443/tasks/?name=shared_pipecheck_20210525T115157Z*
+
+This view displays a short summary of tasks, its statuses and progress.
+For example, a line of the summary table shown in the fig 4.
+
+.. image:: ../_images/TaskSummaryTaskView.jpg
+   :width: 6.5in
+   :height: 0.43056in
+
+Fig 4. Example of the task summary on the tasks view
+
+In this line the first column is the task id in the PanDA system linked
+to a task detailed view. The second column provides the task name. There
+is a message displayed here: “insufficient inputs are ready. 0 files
+available, 1*1 files required” this means that not all pseudo inputs
+(data ids) for this task are released because the previous steps are not
+yet finished and currently this task has no unprocessed inputs. The
+third column shows the task status and number of pseudo inputs (data
+ids) registered for this task. Each data input corresponds to a unique
+job to be submitted in the computation cluster. In this case the task
+unites 1180 jobs. The third column shows the overall completion progress
+(84% or 1001 jobs) and the failure rate (9% or 64 jobs).
+
+Following columns used for the system debug.
+
+Jobs progress
+-------------
+
+Clicking on the task id or its name on the tasks view the detailed
+information is loaded, as shown on the fig. 5:
+
+.. image:: ../_images/Fig5TaskDetail.jpg
+   :width: 5.95313in
+   :height: 4.4446in
+
+Fig 5. Task details
+
+Here one can see several tables, one of the most important is the jobs
+summary. In this table all jobs of the task are counted and grouped by
+their statuses. Since PanDA uses late jobs generation, a job is
+generated only when the next available input is released.
+
+There are two retry filtration modes supported: drop and non drop. They
+could be switched by clicking the correspondent link in the table head.
+The drop mode hides all failed jobs which were successfully retried and
+shows only failures which are hopeless or not yet addressed by the retry
+module. The drop mode is the default one. The non drop mode shows every
+failure regardless if they were retried. It could be directly specified
+in the query URL as follows::
+
+    https://usdf-panda-bigmon.slac.stanford.edu:8443/task/<taskid>/?mode=nodrop
+
+Logs access
+-----------
+
+PanDA monitor provides central access to logs generated by running jobs.
+A log becomes accessible when a job is in the final state - e.g.
+finished or failed. In the IDF deployment every log is transferred to
+the object store and then available for download from there. There are 2
+kinds of job logs available: the Rubin software output and the Pilot log
```

Comment:
Maybe instead of available, say managed by PanDA.  The per-Quantum log file (i.e., json file) also goes to the butler repo (butler query-datasets --show-uri <butler path> --collections <run collection> '*_log')

Reply:
I'm trying to think of a couple example of errors where one would find them in the pilot log but not the output log.  bad payload command line.  I can't think of another now.

Reply:
This section mainly says 'PanDA monitor provides central access to logs generated by running jobs.' PanDA monitor doesn't manage the logs, it just provides links to the logs.
I added description about pilot logs, which are mainly for administrators to understand the issues when something doesn't work. Users don't need to dig it.